SECURITY.md

```diff
@@ -0,0 +1,240 @@
+  - /contributing/security-policy/
+---
+
+# Knative security policy
+
+## Introduction
+
+Security is core to our values, and we value the input of developers, operators,
+security researchers, and others acting in good faith to help us maintain a high
+standard for the security and privacy for our users, and their users. This
+includes encouraging responsible vulnerability research and
+[coordinated disclosure](#coordinated-vulnerability-disclosure). This policy
+sets out our definition of good faith in the context of finding and reporting
+vulnerabilities, as well as what you can expect from us in return.
+
+_If you have identified a vulnerability in Knative, please do not post an issue
+in the public issue tracker. Instead, please follow the coordinated disclosure
+policy described in this document._
+
+## Expectations
+
+When working with us according to this policy, you can expect us to:
+
+- Extend [Safe Harbor](#safe-harbor) for your vulnerability research that is
+  related to this policy;
+- Work with you to understand and validate your report, including a timely
+  initial response to the submission;
+- Work to remediate discovered vulnerabilities in
+  [supported versions](#scope-and-supported-versions) in a timely manner;
+- [Coordinate disclosure](#coordinated-vulnerability-disclosure) with vendors in
+  our CVD program;
+- Publicly disclose accepted vulnerabilities in a timely manner; and
+- Recognize your contribution to improving our security if you are the first to
+  report a unique vulnerability, and your report triggers a code or
+  configuration change.
+
+## Ground Rules
+
+To encourage vulnerability research and to avoid any confusion between
+good-faith hacking and malicious attack, we ask that you:
+
+- Play by the rules. This includes following this policy, as well as any other
+  relevant agreements, particularly the Knative
+  [Code of Conduct](https://knative.dev/community/contributing/code-of-conduct/).
+  If there is any inconsistency between this policy and any other relevant
+  terms, the terms of this policy will prevail;
+- Report any vulnerability you’ve discovered promptly;
+- Avoid violating the privacy of others, disrupting out-of-scope systems,
+  destroying data, and/or harming user experience;
+- Use only the [Official Channels](#reporting-a-vulnerability) to discuss
+  vulnerability information with us;
+- Keep the details of any discovered vulnerabilities confidential until they are
+  fixed, according to our
+  [Coordinated Disclosure Policy](#coordinated-vulnerability-disclosure);
+- Perform testing only on in-scope systems, and respect systems and activities
+  which are out-of-scope;
+- If a vulnerability provides unintended access to data: Limit the amount of
+  data you access to the minimum required for effectively demonstrating a Proof
+  of Concept; and cease testing and submit a report immediately if you encounter
+  any user data during testing, such as Personally Identifiable Information
```

Comment:
Does it make sense to include links for these (PII and PHI) so it’s clear to the reader. For instance, this one comes up for PII from US government: https://www.dol.gov/general/ppii but perhaps there’s better?